properties.py

```diff
@@ -264,6 +264,16 @@ def update_type(self, context):
         name="AOV name",
         description="Name of the built in AOV")
 
+    denoise_aov = BoolProperty(
+        name="Format AOV for denoising",
+        description="If checked this pass will be export as an individual file and properly formatted for use by the denoise utility",
+        default=False)
+
+    exclude_from_multi = BoolProperty(
```

Comment:
Yeah would be nice if the pass editor was more flexible!


On Wed, May 11, 2016 at 4:14 PM, Jon D <notifications@github.com> wrote:

> In properties.py
> <https://github.com/bsavery/PRMan-for-Blender/pull/258#discussion_r62942839>
> :
>
> > @@ -264,6 +264,16 @@ def update_type(self, context):
> >          name="AOV name",
> >          description="Name of the built in AOV")
> >
> > +    denoise_aov = BoolProperty(
> > +        name="Format AOV for denoising",
> > +        description="If checked this pass will be export as an individual file and properly formatted for use by the denoise utility",
> > +        default=False)
> > +
> > +    exclude_from_multi = BoolProperty(
>
> Or I just got carried away...... Not quite sure which
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/bsavery/PRMan-for-Blender/pull/258/files/57beb09857478d08591b5b665692f756d9eb80f3#r62942839>
>



-- 
brian.savery@gmail.com
508-274-8700